app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -28,7 +28,7 @@ class Api::V2::RepositoriesController < Api::V2::ApiController
       param :url, String, :desc => N_("repository source url")
       param :gpg_key_id, :number, :desc => N_("id of the gpg key that will be assigned to the new repository")
       param :unprotected, :bool, :desc => N_("true if this repository can be published via HTTP")
-      param :content_type, String, :required => true, :desc => N_("type of repo (either 'yum', 'puppet', 'docker', or 'ostree')")
+      param :content_type, RepositoryTypeManager.creatable_repository_types.keys, :required => true, :desc => N_("type of repo (either 'yum', 'puppet', 'docker', or 'ostree')")
```

Comment:
Nah, I should probably actually remove it from the description. If you use an enum as the type, you get the proper doc/validation:

![screen shot 2016-02-23 at 9 05 08 am](https://cloud.githubusercontent.com/assets/86290/13253926/c0eade48-da0c-11e5-8af1-5b250cc8e68d.png)